pkg/js/util/exceptions.ts

```diff
@@ -125,7 +145,7 @@ const createInvalidConditionNameInParameterError = (
   relationName: string,
   conditionName: string,
 ) => {
-  const { errors, lines, lineIndex, symbol } = props;
+  const { errors, lines, lineIndex, symbol, module, file } = props;
   // const rawLine = lines[lineIndex];
```

Comment:
Can we remove the commented out lines?

Reply:
I think we can, will review

Reply:
Done in 9cc4c6e9156a7468341b2eb127cdf20736b5bf21